### ISSUE TITLE
Feature request: Bedrock Knowledge Bases Connector

### ISSUE THREAD

**Issue by HQarroum** [awslabs/project-lakechain#8]
### Use case

Integrate Bedrock Knowledge Bases with Lakechain to allow customers to publish their documents in a Bedrock Knowledge Base

**Comment by moltar** [comment]
While reading the docs I expected this to be one of the integrations and was quite surprised that it was not there. In fact, I kept looking for it, and even checked the code base to see if perhaps it was just undocumented.

**Comment by HQarroum** [comment]
Bedrock Knowledge Base takes its input documents from S3, it is a managed end-to-end service. We created this feature request as a mean to later integrate with Bedrock KB, but I don't think it is immediately possible today.

If you need to process documents *before* they get indexed into Bedrock KB, you can do so with Lakechain using the [S3 Connector](https://awslabs.github.io/project-lakechain/triggers/s3-event-trigger/) and store your processed documents into the S3 bucket associated with your knowledge base.

But if we're missing something, let us know your exact use-case and the Bedrock KB API you'd like Lakechain to create an integration with.

**Comment by HQarroum** [comment]
Makes sense. We will provide an example for an integration with Bedrock Knowledge Bases.
